content/references/http-websocket-apis/admin-api-methods/logging-and-data-management-methods/node_to_shard.ja.md

```diff
@@ -0,0 +1,122 @@
+---
+html: node_to_shard.html
+parent: logging-and-data-management-methods.html
+blurb: レジャーストアからシャードストアにデータをコピーします。
+labels:
+  - データ保持
+---
+# node_to_shard
+[[ソース]](https://github.com/ripple/rippled/blob/develop/src/ripple/rpc/handlers/NodeToShard.cpp "Source")
+
+`{{currentpage.name}}`メソッドは、レジャーストアから[シャードストア](history-sharding.html)へのデータコピーを管理します。データコピーの開始、停止、状態チェックが可能です。
+
+_`{{currentpage.name}}`メソッドは、権限のないユーザーには実行できない[管理メソッド](admin-api-methods.html)です。_
+
+
+### リクエストフォーマット
+
+リクエストフォーマット例:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+    "command": "{{currentpage.name}}",
+    "action": "start"
+}
+```
+
+*JSON-RPC*
+
+```json
+{
+    "method": "{{currentpage.name}}",
+    "params": [{
+        "action": "start"
+    }]
+}
+```
+
+*Commandline*
+
+```sh
+#Syntax: {{currentpage.name}} start|stop|status
+rippled {{currentpage.name}} start
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+リクエストは、以下のパラメータを含みます:
+
+| `項目`  | 型   | 説明
+|:---------|:-------|:---------------------------------------------------------|
+| `action` | String | どのような動作をさせるかによって、 `start`、`stop` または `status` のいずれかを指定します。 |
```

Comment:
For consistency with most previously translated pages, we should probably use 文字列 rather than "String" in the type columns too, right?

```suggestion
| `action` | 文字列 | どのような動作をさせるかによって、 `start`、`stop` または `status` のいずれかを指定します。 |
```